src/assets/scripts/client/aircraft/AircraftModel.js

```diff
@@ -592,12 +593,17 @@ export default class AircraftModel {
         let assignedAltitude = this.mcp.altitude;
         let flightPlanAltitude = this.fms.flightPlanAltitude;
 
+
         if (assignedAltitude === INVALID_NUMBER) {
             assignedAltitude = '-';
+        } else {
+            assignedAltitude *= UNIT_CONVERSION_CONSTANTS.FT_FL;
```

Comment:
Knew you'd say that. 😉 I had already made that change before I figured out it won't work out that way. If we start by setting it to `mcp.altitude * UCC.FT_FL`, we can't ask `if (assignedAltitude === INVALID_NUMER)`. We _could_ ask `if (assignedAltitude === INVALID_NUMBER * UCC.FT_FL)`, but that seemed rather.... odd. At least when compared to adding an else.